NFe.Servicos/ServicosNFe.cs

```diff
@@ -723,14 +723,13 @@ private RetornoRecepcaoEvento RecepcaoEvento(int idlote, List<evento> eventos, S
         /// <param name="tipoDocumento">Tipo de documento a ser consultado</param>
         /// <param name="documento">Documento a ser consultado</param>
         /// <returns>Retorna um objeto da classe RetornoNfeConsultaCadastro com o retorno do serviço NfeConsultaCadastro</returns>
-        public RetornoNfeConsultaCadastro NfeConsultaCadastro(string uf, ConsultaCadastroTipoDocumento tipoDocumento,
-            string documento)
+        public RetornoNfeConsultaCadastro NfeConsultaCadastro(string uf, ConsultaCadastroTipoDocumento tipoDocumento, string documento)
         {
             var versaoServico = ServicoNFe.NfeConsultaCadastro.VersaoServicoParaString(_cFgServico.VersaoNfeConsultaCadastro);
 
             #region Cria o objeto wdsl para consulta
 
-            var ws = CriarServico(ServicoNFe.NfeConsultaCadastro);
+            var ws = CriarServico(ServicoNFe.NfeConsultaCadastro, uf);
```

Comment:
Aqui não mudaria o comportamento atual?

Reply:
Bom, compilei aqui e atualizei alguns clientes, até o momento tudo normal, não alterando o comportamento atual da Lib.
Pois se o argumento uf for nullo no assinatura do Método, quando for executado o ObterUrlServico vai pegar por padrao a Uf da classe ConfiguracaoServico.
Está forma foi a maneira que pensei rapidamente para simplesmente Consultar dados do CNPJ de outros estados